backend/server.py

```
@@ -5863,12 +5863,49 @@ async def autonomy_status() -> dict[str, object]:
+        pass
+    try:
+        from services.self_bootstrap import _find_self_company
+        existing = await _find_self_company()
+        if existing is not None:
+            self_bootstrap_status["company_id"] = existing.id
+            self_bootstrap_status["onboarding_status"] = existing.onboarding_status
+            self_bootstrap_status["domain"] = existing.domain
+        else:
+            self_bootstrap_status["company_id"] = None
+            self_bootstrap_status["onboarding_status"] = "not_started"
+    except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)